control_panel_api/urls.py

```diff
@@ -5,14 +5,14 @@
 
 from control_panel_api import views
 
-
 router = routers.DefaultRouter()
 router.register(r'apps', views.AppViewSet)
 router.register(r'apps3buckets', views.AppS3BucketViewSet)
 router.register(r'users3buckets', views.UserS3BucketViewSet)
 router.register(r'groups', views.GroupViewSet)
 router.register(r's3buckets', views.S3BucketViewSet)
 router.register(r'users', views.UserViewSet)
+router.register(r'appusers', views.AppUserViewSet)
```

Comment:
I think the general rule on trying to keep things alphabetically sorted applies - unless there is a specific reason why order is important. Don't think it's the case in this instance (but as I said, I'd prefer the model/entpoints to be `UserApp`/`userapps` as in my view it's the user that is the subject managing the apps in this case).